fem/fespace.hpp

```diff
@@ -1010,6 +1098,115 @@ class QuadratureInterpolator
                       const int eval_flags);
 };
 
+/** @brief A class that performs interpolation from an E-vector to quadrature
+    point values and/or derivatives (Q-vectors) on the faces. */
+/** An E-vector represents the element-wise discontinuous version of the FE
+    space and can be obtained, for example, from a GridFunction using the
+    Operator returned by FiniteElementSpace::GetElementRestriction().
+
+    The target quadrature points in the elements can be described either by an
+    IntegrationRule (all mesh elements must be of the same type in this case) or
+    by a QuadratureSpace. */
+class FaceQuadratureInterpolator
+{
+private:
+   FaceType type;
```

Comment:
I get warnings about this being unused when compiling with clang. However, if I delete this line I get compilation errors. Should look into this.